js/nested-for-loops.js

```diff
@@ -5,11 +5,25 @@
  * @returns {string}
  */
 function nestedForLoops (depth, width = depth) {
-  let result = "\n";
+  let result = "\n"; // "NEW LINE"
 
-  // Write code here
+  let arr = [];
+
+  for(a=0; a<depth; a++){
```

Comment:
when declaring a variable in a for loop make sure you put `let` in front of it.
`for (let a = 0; a < depth; a++) {`